packages/app-store/routing-forms/lib/formSubmissionUtils.test.ts

```diff
@@ -151,6 +193,58 @@ describe("_onFormSubmission", () => {
         },
       });
     });
+
+    it("should call WorkflowService.scheduleFormWorkflows for FORM_SUBMITTED_NO_EVENT workflows", async () => {
+      const mockWorkflows = [
+        {
+          id: 2,
+          name: "Form Follow-up Workflow",
+          userId: 1,
+          teamId: null,
+          trigger: WorkflowTriggerEvents.FORM_SUBMITTED_NO_EVENT,
+          time: 30,
+          timeUnit: "MINUTE",
+          steps: [
+            {
+              id: 2,
+              action: WorkflowActions.EMAIL_ATTENDEE,
+              sendTo: null,
+              reminderBody: "Follow up on your form submission",
+              emailSubject: "Follow Up",
+              template: WorkflowTemplates.CUSTOM,
+              verifiedAt: new Date(),
+              includeCalendarEvent: false,
+              numberVerificationPending: false,
+              numberRequired: false,
+            },
+          ],
+        },
+      ];
+
+      vi.mocked(getAllWorkflowsFromRoutingForm).mockResolvedValueOnce(mockWorkflows as any);
```

Comment:
Test are failing because of https://github.com/calcom/cal.com/pull/23716#discussion_r2375030109